src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -387,6 +387,10 @@ func chained(rootCA string, persistent bool) (http.RoundTripper, error) {
 			ClientSessionCache: clientSessionCache,
 		},
 	}
+	if persistent {
```

Comment:
I'm just saying that `persistent` flag to the `chained` method just specifies whether or not to set `DisableKeepAlives` in the `http.Transport`. The effectively governs whether or not the transport will add the `Connection: close` header. 

All I'm saying is I'm not sure the inclusion or lack of `Connection: close` (the `if persistent`) makes sense as the criteria for deciding whether or not to set a max idle time. It seems like we might just want to always set it.